app/controllers/spotlight_search/concerns/exportable.rb

```diff
@@ -0,0 +1,6 @@
+module SpotlightSearch
```

Comment:
You are planning to use this concern in future?

Reply:
This concern was supposed to be used when we would give user the ability to select columns and stuff. It would be used in the future. I can remove it for now.